azure-pipelines.yml

```diff
@@ -127,3 +127,41 @@ jobs:
   - script: echo $(NODE_OPTIONS)
   - script: "scripts/swagger-to-sdk.sh Azure/$(AZURE_SDK_REPO) -v $(AZURE_SDK_PARAMS)"
     displayName: "Swagger to SDK script"
+
+- job: "NetworkValidation"
+  condition: and(startsWith(variables['System.PullRequest.TargetBranch'], 'network-'), endsWith(variables['System.PullRequest.TargetBranch'], '-release'))
```

Comment:
@number213 Could we also put ` continueOnError: true`? So it will not block other PRs. As soon as it merged, we will monitor this CI for few weeks and if it works fine we will remove `continueOnError: true`.

Reply:
@sergey-shandar done